src/client/components/header/index.js

```diff
@@ -7,6 +7,22 @@ import upload from './assets/upload.png';
 import userprofile from './assets/userprofile.png';
 
 class header extends React.Component {
+  Userlogout() {
+    let token = localStorage.getItem('token');
+
+    fetch('/api/logout', {
+      method: 'delete',
+      headers: {
+        'Content-Type': 'application/json',
+        'token': token,
```

Comment:
we have an Authorization field in the [header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Authorization). we have to use that to transfer the token.